src/init.cpp

```diff
@@ -369,6 +369,7 @@ void SetupServerArgs()
             "(default: 0 = disable pruning blocks, 1 = allow manual pruning via RPC, >=%u = automatically prune block files to stay under the specified target size in MiB)", MIN_DISK_SPACE_FOR_BLOCK_FILES / 1024 / 1024), false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-reindex", "Rebuild chain state and block index from the blk*.dat files on disk", false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-reindex-chainstate", "Rebuild chain state from the currently indexed blocks. When in pruning mode or if blocks on disk might be corrupted, use full -reindex instead.", false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-mitigatespectre", strprintf("Attempt to enable per process spectre mitigations (default: %u, defaults to disabled if -disablewallet is set)", DEFAULT_MITIGATE_SPECTRE), true, OptionsCategory::OPTIONS);
```

Comment:
The default-exception language here doesn't match elsewhere.

Maybe something like:
```C++
    gArgs.AddArg("-mitigatespectre", strprintf("Attempt to enable per process spectre mitigations (default: %u%s)", DEFAULT_MITIGATE_SPECTRE, (
#ifdef ENABLE_WALLET
    DEFAULT_MITIGATE_SPECTRE ? "0 or 1 if -disablewallet set" : "0"
#else
    strprintf("%u", DEFAULT_MITIGATE_SPECTRE)
#endif
    )), true, OptionsCategory::OPTIONS);
```